meson.build

```diff
@@ -75,60 +75,50 @@ configinc = include_directories('.')
 cc = meson.get_compiler('c')
 dl_dep = cc.find_library('dl', required : false)
 
+# The libdrm package is required by all backends
 libdrm_dep = dependency('libdrm', version : '>= 2.4.60')
 
 backends = ''
-WITH_DRM = not get_option('disable_drm')
-if WITH_DRM
-  backends += 'drm'
+WITH_DRM = false
+if libdrm_dep.found() and not get_option('with_drm').disabled()
```

Comment:
libdrm_dep cannot ever be not-found, so this one is a false choice

Reply:
Correct libdrm_dep.found is always true. I've added it for symmetry with the other code-paths and it's not causing any issues - be that confusing meson or adding any measurable overhead.